examples/customresources/demos/simplemodule/module.go

```diff
@@ -47,11 +48,16 @@ func mainWithArgs(ctx context.Context, args []string, logger logging.Logger) err
 		return err
 	}
 
-	// This will block (leaving the module running) until the context is cancelled.
-	// The utils.ContextualMain catches OS signals and will cancel our context for us when one is sent for shutdown/termination.
-	<-ctx.Done()
-	// The deferred myMod.Close() will now run.
-	return nil
+	for {
+		if err := ctx.Err(); err != nil {
+			return nil
+		}
+		time.Sleep(time.Second)
+		logger.Debugw("debug message", "myKey", "myVal")
+		logger.Infow("info message", "myKey", "myVal")
+		logger.Warnw("warn message", "myKey", "myVal")
+		logger.Errorw("error message", "myKey", "myVal")
+	}
```

Comment:
These logs now look like:
```
2023-12-13T18:01:54.941-0500    ERROR   robot_server.simple-module      simplemodule/module.go:59 "error message" {"module_log_ts": Dec 13 23:01:54, myKey: myVal}
2023-12-13T18:01:55.943-0500    INFO    robot_server.simple-module      simplemodule/module.go:57 "info message" {"module_log_ts": Dec 13 23:01:55, myKey: myVal}
2023-12-13T18:01:55.944-0500    WARN    robot_server.simple-module      simplemodule/module.go:58 "warn message" {"module_log_ts": Dec 13 23:01:55, myKey: myVal}
```
When received by a non-debug server.

And like:

```
2023-12-11T18:15:13.928-0500	INFO	robot_server.process.SimpleModule_/Users/benji.rewis/Code/rdk/examples/customresources/demos/simplemodule/simplemodule.StdOut	pexec/managed_process.go:244	
\_ 2023-12-11T23:15:13.928Z	INFO	simple-module	simplemodule/module.go:57	info message
2023-12-11T18:15:13.928-0500	INFO    robot_server.process.SimpleModule_/Users/benji.rewis/Code/rdk/examples/customresources/demos/simplemodule/simplemodule.StdOut	pexec/managed_process.go:244	
\_ 2023-12-11T23:15:13.928Z	WARN	simple-module	simplemodule/module.go:58	warn message
2023-12-11T18:15:13.928-0500	INFO	robot_server.process.SimpleModule_/Users/benji.rewis/Code/rdk/examples/customresources/demos/simple-module/simplemodule.StdOut	pexec/managed_process.go:244	
\_ 2023-12-11T23:15:13.928Z	ERROR	simple-module	simplemodule/module.go:59	error message
```
When sent to the module's STDOUT due to an inactive or unhealthy gRPC connection.

Reply:
Looks great! I left a comment in the string formatting below. And thanks for the examples so I didn't have to guess if the string format was as I expected.

Reply:
Updated those examples further to show what your suggesting formatting is now producing (no quote around the message and `\t` between caller and message).